setup/scRNA_qc/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "scrna_qc"
-version = "1.0.12"
+version = "1.0.13"
```

Comment:
if you're just changing the build, I think it isn't mandatory to change the version.
if it's related to a wdl, don't forget to update globals in terra_pipeline repo

Reply:
you are right but I don't want to change it back to avoid issues in next versions